index.js

```diff
@@ -27,7 +27,7 @@ const loader = function(content)
   }
 
   if (config.vars) {
-    Object.assign(vars, config.config);
+      Object.assign(vars, JSON.parse(config.vars));
```

Comment:
The `JSON.parse()` shouldn't be necessary here. But yeah, `config.config` was definitely a typo, good catch! :)